tests/testsuite/cargo_add/add_no_vendored_package_with_vendor/mod.rs

```diff
@@ -1,44 +1,12 @@
 use cargo_test_support::compare::assert_ui;
-use cargo_test_support::paths;
 use cargo_test_support::prelude::*;
-use cargo_test_support::ProjectBuilder;
+use cargo_test_support::Project;
 
 use cargo_test_support::curr_dir;
 
 #[cargo_test]
 fn case() {
-    let project = ProjectBuilder::new(paths::root().join("in"))
-        .file(
-            ".cargo/config.toml",
-            r#"[source.crates-io]
-replace-with = "vendored-sources"
-
-[source.vendored-sources]
-directory = "./vendor"
-"#,
-        )
-        .file("vendor/aa/src/lib.rs", "")
-        .file("vendor/aa/.cargo-checksum.json", "{\"files\":{}}")
-        .file(
-            "vendor/aa/Cargo.toml",
-            r#"[workspace]
-
-[package]
-name = "aa"
-version = "0.0.0"
-"#,
-        )
-        .file("src/lib.rs", "")
-        .file(
-            "Cargo.toml",
-            r#"[workspace]
-
-[package]
-name = "cargo-list-test-fixture"
-version = "0.0.0"
-"#,
```

Comment:
Was it intentional that `aa` wasn't carried over?  Seems like its needed to get the original error message.

Reply:
This imitated the `in/` format to create `aa` (in the vendor directory) instead of ProjectBuilder.